receiver/mongodbatlasreceiver/config.go

```diff
@@ -88,7 +89,14 @@ type OrgConfig struct {
 	ID string `mapstructure:"id"`
 }
 
-func (pc *ProjectConfig) populateIncludesAndExcludes() *ProjectConfig {
+// AccessLogsConfig is the configuration options for access logs collection
+type AccessLogsConfig struct {
+	Projects     []ProjectConfig `mapstructure:"projects"`
```

Comment:
Do you have thoughts on having a maximum number of access logs we will go back to i.e. something similar to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/a4e4fe4af4f7c7fe0deb8720c92db1b5801461a6/receiver/mongodbatlasreceiver/config.go#L74

where we give users a way to configure a hard limit of the number of logs we will go back into the history for?

Reply:
Seems reasonable.